Layout_index_v1.html

```diff
@@ -18,63 +18,86 @@
 		<div class="grid-container">
 			<div>
 				
-		<div class="header">Home Page</div>
+	    <div class="header"><img src="images/LOGO2.jpg" alt="iMages"></div>
 			
 		<div>
 			
 	<!--This class is for my navigation/menu-->
     <div class="nav">
 		<ul>
-			<li><a href="index_v1.html">Home | </a></li>
-			<li><a href="CommunityService.v1.html">Community service | </a></li>
+			<li><a href="index.v2.html">Home | </a></li>
+			<li><a href="CommunityService.v2.html">Community service | </a></li>
 			<li><a href>Outdoor activities |</a>
 			    <ul>
-					<li><a href="indexpage2.v1.html">Adrenaline Forest and Te Rauparaha Arena</a>
-					<li><a href="indexpage3.v1.html">Rangituhi Walk</a>
-					<li><a href="indexpage4.v1.html">Wainuiomata Camp pt1</a>
-					<li><a href="indexpage5.v1.html">Wainuiomata Camp pt2</a>
+					<li><a href="indexpage2.v2.html">Adrenaline Forest and Te Rauparaha Arena</a>
+					<li><a href="indexpage3.v2.html">Rangituhi Walk</a>
+					<li><a href="indexpage4.v2.html">Wainuiomata Camp pt1</a>
+					<li><a href="indexpage5.v2.html">Wainuiomata Camp pt2</a>
 				  </ul>
 				  </li>
 			     <li><a href="Contact.v1.html">Contact |</a>
 			    </ul> 
-            </div>		
-	
+            </div>	
 
 			<div>
 			<div class="box2"></div>
 			
 			<div class="box3"></div>
 			</div>
-			
-			<div class="sidebar">sidebar</div>
 		
-		
-			<div class="content">	
-			   <div class="scroll-area">
-				   
+		   
+		<div class="content">	
+			<div class="scroll-area">
+			
+				<!-- Setup Class to hold and control Radio input Buttons to move each image. -->
+        <div class="slider">
+		   <div class="images">
+		      <input type="radio" name="slide" id="img1" checked> <!-- Radio is the style of the input buttons. Checked activates the CSS -->
+              <input type="radio" name="slide" id="img2"> <!-- ID Controls each separate image through the CSS using a # -->
+			  <input type="radio" name="slide" id="img3">
+			  <input type="radio" name="slide" id="img4">
+			  
+			  <img src="images/iMAGE.jpg" class="m1" alt="iMAGE"> <!-- Class controls  each separate image through CSS -->
+			  <img src="images/iMAGE.jpg" class="m2" alt="iMAGE">
+			  <img src="images/iMAGE.jpg" class="m3" alt="iMAGE">
+			  <img src="images/iMAGE.jpg" class="m4" alt="iMAGE">
+			</div>
+			
+			<!-- Setup Class to hold and control Radio input Buttons to move each image-->
+			<div class="dots">
+			   <label for="img1"></label>
+			   <label for="img2"></label>
+			   <label for="img3"></label>
+			   <label for="img4"></label>
+			</div>

```

Comment:
in my second version for my layout, I added an Image carousel. As I thought it would look good at the top of the content